src/main/java/vane/MultiplicationTable.java

```diff
@@ -0,0 +1,19 @@
+package main.java.vane;
+
+import java.util.Scanner;
+
+/**
+ * Created by Vane Gharakhanian on Sep, 2019
+ */
+public class MultiplicationTable {
+    public static void main(String args[]) {
+        Scanner s = new Scanner(System.in);
+        System.out.print("Please Enter number:");
+        int n = s.nextInt();
+        int i = 1;
+        while (i < 10) {
+            System.out.println(n + " x " + i + " = " + n * i);
```

Comment:
konkret estex, repeat-i qanaky gites, arjer for-ov anel, bayc de senc el klni )

Reply:
skzbic for ov ei grel heto pokheci :) kani vor mnacac tegherum elei ogtagorcel for-e mtaceci es meke while ogtagorcem :D